Task 2/Task 2.2/Task2.2.1.GameLogic.Interfaces/IMap.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Task2._2._1.GameLogic.Interfaces
+{
+    public interface IMap
+    {
+        int Width { get; }
+        int Height { get; }
+        IPlayer Player { get; }
+        Dictionary<Point, IBarrier> Barriers { get; }
+        Dictionary<Point, IBonus> Bonuses { get; }
+        Dictionary<Point, IEnemy> Enemies { get; }
+        Dictionary<Point, IGameObject> GameObjects { get; }
```

Comment:
A 2d array of collections of game objects would be a better fit (it would check boundaries and could hold multiple objects in a single cell, unless you need to separate them into different layers)